File: 02-greeter/greeter.rs
// Simple Greeter program in Rust
// Created by M.V.Harish Kumar on 17-10-2023
use std::io::{self, Write};

fn main() {
    let mut name = String::new();

    print!("Please enter your name: ");
    io::stdout().flush().unwrap();

    io::stdin()
        .read_line(&mut name)
        .expect("Failed to read line");
    name.pop();

    println!("Hello, {name}! Hope you're fine");
}

File: 02-greeter/Greeter.cs
// Simple Greeter program in C#
// Created by M.V.Harish Kumar on 06-01-2024
using System;

namespace Greeter
{
    class Greeter
    {
        static void Main(string[] args)
        {
            Console.Write("Please enter your name: ");
            string name = Console.ReadLine();
            Console.WriteLine($"Hello, {name}! Hope you're fine");
        }
    }
}